src/Helpers.php

```diff
@@ -868,22 +873,23 @@ public static function imagecreatefrombmp($filename, $context = null)
      *  - curl: if allow_url_fopen is false and curl is available
      *
      * @param string $uri
-     * @param resource $context (ignored if curl is used)
+     * @param resource $context
      * @param int $offset
-     * @param int $maxlen (ignored if curl is used)
+     * @param int $maxlen
      * @return string[]
      */
     public static function getFileContent($uri, $context = null, $offset = 0, $maxlen = null)
     {
         $content = null;
         $headers = null;
         [$protocol] = Helpers::explode_url($uri);
-        $is_local_path = ($protocol === "" || $protocol === "file://");
+        $is_local_path = in_array(strtolower($protocol), ["", "file://", "phar://"], true);
```

Comment:
Checking for `""` here should be unnecessary now.

Reply:
Since it's a public method I left that check in there for people calling the method directly from their own code. Someone actually using this method that way seems unlikely, but with a large enough user base ...

Reply:
Good point.